datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/service_checks.py

```diff
@@ -9,18 +9,42 @@
 from six import string_types
 
 from ....compat import JSONDecodeError
-from ....utils import file_exists, read_file
+from ....utils import file_exists, read_file, write_file
 from ...constants import get_root
 from ...utils import get_valid_integrations, load_manifest, parse_version_parts
 from ..console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success
 
 REQUIRED_ATTRIBUTES = {'agent_version', 'check', 'description', 'groups', 'integration', 'name', 'statuses'}
+SERVICE_CHECK_NAMES = ['ok', 'warning', 'critical', 'unknown']
+
+root = get_root()
```

Comment:
This isn't set until a command is called, e.g. this will always be `None`

Reply:
@ofek Fixed